.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
```

Comment:
You can call this workflow whatever you want here?

Reply:
I'll leave this up to @jacobstanley as he's better in naming things than I am 😆

Reply:
CI is allg 😎